core/multiplayer/components/resume-story-form.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
+
+import { resumeStoryInstance } from '../api-client'
+import { StoryContext } from 'pages/[story]/[[...chapter]]'
+import styles from 'public/styles/multiplayer/ResumeStory.module.scss'
+
+const ResumeStory: React.FC = ({ children = 'Resume an existing story' }) => {
+    const { persistor, config } = React.useContext(StoryContext)
+    const instanceId = React.useRef<HTMLInputElement>(null)
+    const player1 = React.useRef<HTMLInputElement>(null)
+    const player2 = React.useRef<HTMLInputElement>(null)
+
+    return (
+        <div className={styles.resume}>
+            <form
+                className={styles.resumeForm}
+                onSubmit={async (e) => {
+                    e.preventDefault()
+                    const isPlayer1 = player1.current.checked ? 'player1' : null
+                    const isPlayer2 = player2.current.checked ? 'player2' : null
+                    if (!isPlayer1 && !isPlayer2) {
+                        alert('You must select which player to resume.')
+                        return false
+                    }
+
+                    persistor.flush().then(() => {
+                        persistor.pause()
+                        persistor.purge().then(() => {
+                            persistor.persist()
+                        })
+                    })
```

Comment:
Try this out in Reset Game as well as whenever the code hits the `getStoryInstance` endpoint.

Reply:
Nah people still recommend fully resetting the browser but this is fine for this case, probably?